app/services/medium_article_retrieval_service.rb

```diff
@@ -17,13 +17,23 @@ def call
     reading_time = page.at("meta[name='twitter:data1']")["value"]
     author = page.at("meta[name='author']")["content"]
     author_image = page.at("img[alt='#{author}']")["src"]
+    published_time = page.at("meta[property='article:published_time']")["content"]
 
     {
       title: title,
       author: author,
       author_image: author_image,
       reading_time: reading_time,
+      publication_date: publication_date(published_time),
       url: url
     }
   end
+
+  private
+
+  def publication_date(published_time)
+    Date.parse(published_time).strftime("%b %-d, %Y")
```

Comment:
This is better expressed as:

`Time.zone.parse()`

I think `Date.parse` ignores the timezone which might be a problem around the international date time, it also helps in case we want to display the full time in the future.

Reply:
You're right, I completely forgot about timezone.